charts/cass-operator/templates/configmap.yaml

```diff
@@ -29,15 +28,20 @@ data:
     images:
       system-logger: "k8ssandra/system-logger:v1.10.0"
       config-builder: "datastax/cass-config-builder:1.0.4-ubi7"
+{{- if .Values.image.repositoryOverride }}
+    {{- with .Values.image.repositoryOverride }}
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+{{- end }}
 {{- if .Values.image.registryOverride }}
-    {{ indent 2 (cat "imageRegistry:" .Values.image.registryOverride) }}
+    imageRegistry: {{ .Values.image.registryOverride }}
 {{- end }}
 {{- with .Values.imagePullSecrets }}
     imagePullSecrets:
```

Comment:
As noted in #1404 the ImageConfig API only supports a single pull secret with the field named `imagePullSecret`. Can you make the change here?

Reply:
I'll correct this tomorrow.

Reply:
Changes imagePullSecrets to imagePullSecret.